### ISSUE TITLE
ValueError: Backer is empty - android ndk 29,30

### ISSUE THREAD

**Issue by SajjadPourali** [angr/angr#2456]
Hello All,
I have tried to recompile the `fauxware` example for NDK Android API level 30 and 29, and I have got "ValueError: Backer is empty" error. However, it works perfectly with the level 28 and below.
```
sh-3.2$ ls
fauxware   fauxware.c solve.py
sh-3.2$ aarch64-linux-android30-clang fauxware.c -o fauxware
fauxware.c:29:1: warning: non-void function does not return a value [-Wreturn-type]
}
^
1 warning generated.
sh-3.2$ python3 solve.py
WARNING | 2021-01-03 17:32:21,693 | cle.loader | The main binary is a position-independent executable. It is being loaded with a base address of 0x400000.
Traceback (most recent call last):
  File "/Users/---/Desktop/angr-doc/examples/fauxware/solve.py", line 80, in <module>
    sys.stdout.buffer.write(basic_symbolic_execution())
  File "/Users/---/Desktop/angr-doc/examples/fauxware/solve.py", line 17, in basic_symbolic_execution
    p = angr.Project('fauxware')
  File "/usr/local/lib/python3.9/site-packages/angr/project.py", line 224, in __init__
    self.simos.configure_project()
  File "/usr/local/lib/python3.9/site-packages/angr/simos/linux.py", line 82, in configure_project
    tls_obj = self.project.loader.tls.new_thread()
  File "/usr/local/lib/python3.9/site-packages/cle/backends/tls/__init__.py", line 33, in new_thread
    thread = self._thread_cls(self)
  File "/usr/local/lib/python3.9/site-packages/cle/backends/tls/elf_tls.py", line 93, in __init__
    self.memory.add_backer(self.tp_offset + obj.tls_block_offset, image)
  File "/usr/local/lib/python3.9/site-packages/cle/memory.py", line 180, in add_backer
    raise ValueError("Backer is empty!")
ValueError: Backer is empty!
```
```
sh-3.2$ aarch64-linux-android28-clang fauxware.c -o fauxware
fauxware.c:29:1: warning: non-void function does not return a value [-Wreturn-type]
}
^
1 warning generated.
sh-3.2$ python3 solve.py
WARNING | 2021-01-03 17:33:15,080 | cle.loader | The main binary is a position-independent executable. It is being loaded with a base address of 0x400000.
INFO    | 2021-01-03 17:33:15,099 | pyvex.lifting.libvex | ARM64 front end: branch_etc

WARNING | 2021-01-03 17:33:15,106 | angr.storage.memory_mixins.default_filler_mixin | The program is accessing memory or registers with an unspecified value. This could indicate unwanted behavior.
WARNING | 2021-01-03 17:33:15,106 | angr.storage.memory_mixins.default_filler_mixin | angr will cope with this by generating an unconstrained symbolic variable and continuing. You can resolve this by:
WARNING | 2021-01-03 17:33:15,106 | angr.storage.memory_mixins.default_filler_mixin | 1) setting a value to the initial state
WARNING | 2021-01-03 17:33:15,106 | angr.storage.memory_mixins.default_filler_mixin | 2) adding the state option ZERO_FILL_UNCONSTRAINED_{MEMORY,REGISTERS}, to make unknown regions hold null
WARNING | 2021-01-03 17:33:15,107 | angr.storage.memory_mixins.default_filler_mixin | 3) adding the state option SYMBOL_FILL_UNCONSTRAINED_{MEMORY,REGISTERS}, to suppress these messages.
WARNING | 2021-01-03 17:33:15,107 | angr.storage.memory_mixins.default_filler_mixin | Filling register x29 with 8 unconstrained bytes referenced from 0x401698 (_start_main+0x4 in fauxware (0x1698))
WARNING | 2021-01-03 17:33:15,108 | angr.storage.memory_mixins.default_filler_mixin | Filling register x30 with 8 unconstrained bytes referenced from 0x401698 (_start_main+0x4 in fauxware (0x1698))
WARNING | 2021-01-03 17:33:15,263 | angr.storage.memory_mixins.default_filler_mixin | Filling memory at 0x7fffffffffeff58 with 8 unconstrained bytes referenced from 0x500018 (strcmp+0x0 in extern-address space (0x18))
SOSNEAKYsh-3.2$
```
The binary has uploaded in the following URL:
https://ufile.io/40uhnh9m

Thanks.

**Comment by rhelmot** [comment]
I can't download that link anymore, but I suspect this was fixed by angr/cle#275

**Comment by SajjadPourali** [comment]
Yes, It just fixed.
Thanks.
